src/WordPress/Hooks.php

```diff
@@ -256,6 +256,20 @@ public function getPostRelatedLinks($postId)
             array_push($listofurls, $pageLink);
         }
 
+        // Refresh pagination
+        $total_posts_count = wp_count_posts()->publish;
+        $posts_per_page = get_option('posts_per_page');
+        // Limit to up to 3 pages
+        $page_number_max = min(3, ceil($total_posts_count / $posts_per_page));
+
+        $this->logger->debug("total_posts_count $total_posts_count");
```

Comment:
do we need to keep these debug lines around?

Reply:
> do we need to keep these debug lines around?

I think yes, we don't enable logging by default, so when we need logging it's useful to include.